cli/tests/pcluster/config/test_param_to_cfn.py

```diff
@@ -87,6 +87,11 @@ def test_param_to_cfn_value(mocker, section_definition, param_key, param_value,
         (CLUSTER_SIT, "shared_dir", "test", {"SharedDir": "test"}),
         # (CLUSTER_SIT, "shared_dir", {"ebs": [], "shared_dir": "test"}, {"SharedDir": "test"}),
         # (CLUSTER_SIT, "shared_dir", {"ebs": [{"label": "fake_ebs"}], "shared_dir": "unused_value"}, {}),
+        # ArgsParam
+        (CLUSTER_SIT, "pre_install_args", '"R wget"', {"PreInstallArgs": '\\"R wget\\"'}),
+        (CLUSTER_SIT, "pre_install_args", "'R wget'", {"PreInstallArgs": "'R wget'"}),
```

Comment:
Do we need to add a test case for no `""`, so like `"pre_install_args", "single_argument_no_quotes"`

Reply:
Done